### PR TITLE
Js hw3

### PR DESCRIPTION
**Pull request by emma13z**
Not sure if I have done the "switch" assignment with test grades right. I tried to do it vice versa and didn't manage to display it properly.

image

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    case "FR":
+        console.log("France");
+        break;
+        default:
+    console.log("Unknown country");
+}
+
+let grade = "A";
+switch (grade) {
+    case "A":
```
Comment (HappyJuliana) [inline script.js]: I did it like that 
switch (true) {
    case score >= 90:
        grade = 'A';
        break;

File: script.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+let person2 = tellFortune (3, 'Jack', 'Madrid', 'PR Specialist');
+console.log(person2);
+
+let person3 = tellFortune (4, 'Bob', 'Paris', 'CTO');
+console.log(person3);
+
+// Dog's age exercise //
+
+function calculatedDogAge (humanyears, coefficient=7) {
```
Comment (HappyJuliana) [inline script.js]: As I remember from the condition of our HW, this function should have had only one argument: age of your puppy

File: script.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    return (`Your dog is ${humanyears} years old in dog years`);
+}
+
+let dog1 = calculatedDogAge (4*7);
```
Comment (HappyJuliana) [inline script.js]: let dogAge = x*7;

I included this in the function in order not to repeat every time (6 * 7) or (4 * 7)
And then just called the function and indicated different x

**Review by HappyJuliana — APPROVED** [review]

File: script.js (hunk 1 above)
Comment (emma13z) [reply script.js]: Thanks for the hint! Didn't want just to copy paste and in the end just got lost in all 100500 ways to write this code :)

File: script.js (hunk 2 above)
Comment (emma13z) [reply script.js]: Yeah, but there is also a bonus point, asking to add an extra argument that will change the coefficient. Maybe I misunderstood smth though.
image

File: script.js (hunk 3 above)
Comment (emma13z) [reply script.js]: Good point, thank you!

File: script.js (hunk 2 above)
Comment (HappyJuliana) [reply script.js]: Oh, sorry, I didn’t pay attention to that.
You’re right!

Comment (emma13z) [reply script.js]: np :)

File: script.js (hunk 1 above)
Comment (HappyJuliana) [reply script.js]: Yeah, I understand you
You’re welcome!

File: script.js [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+let quantity = 5;
+let availability = true;
+let totalPrice = availability ? pricePerUnit * quantity : "N/A";
+console.log(pricePerUnit);
+console.log(quantity);
+console.log(availability);
+console.log(totalPrice);
+
+let currentYear = new Date().getFullYear();
+let leapYear = currentYear % 4 === 0 || (currentYear % 400 === 0 && currentYear % 100 === 0) ? "A Leap Year" : "Not A Leap Year";
```
Comment (IvanCeban) [inline script.js:70]: Is this condition necessary?
(currentYear % 400 === 0 && currentYear % 100 === 0)

File: script.js (hunk 1 above)
Comment (IvanCeban) [reply script.js]: Well the idea was viceversa to check the number and give letter as a mark. But technically all is correct :)

**Review by IvanCeban — APPROVED** [review]
Well done!

**Pull request closed without merging**